app/models/file_set.rb

```diff
@@ -1,4 +1,21 @@
 # Generated by hyrax:models:install
 class FileSet < ActiveFedora::Base
   include ::Hyrax::FileSetBehavior
+
+  def language
+    # Future: Use to set language on fileset creation.
+    return try_language(:ocr_language).join("+") if
+      try_language(:ocr_language).present?
```

Comment:
also try_language(:language)?
also check :ocr_language, :language on parent work if blank on FileSet?
Add :ocr_language as a Work and/or FileSet property?  This code still passes without this property, since it's calling `try`, but this was a pumpkin-added field that neither FileSet nor any Work type have currently.